build.xml

```diff
@@ -990,7 +990,7 @@
                          path="net.bobis.jinput.hidraw.HidRawEnvironmentPlugin"/>
             <sysproperty key="purejavacomm.loglevel" value="${purejavacomm.loglevel}"/>
             <jvmarg value="-Xms256m"/>
-            <jvmarg value="-Xmx640m"/>
+            <jvmarg value="-Xmx2048m"/>
```

Comment:
We expect JMRI to be able to run on Raspberry Pi, which has only 1 GB of RAM, which is why the limit is 640 MB (to allow the OS to have some RAM it self). I think this should be reverted, and the problem that causes too much memory to be used to be addressed instead.

Reply:
> But it sure makes things run very slow on the old iMac i'm using. I'll probably have to work with my local copy of build.xml modified most of the time.
> 
> Whether I revert it or not will have no effect on Raspberry Pi users but increasing it sure speeds up development work.
> 

But not reverting it makes it impossible for me to use the Pi as a development box.

But here is the real question: what makes the LokSound decoder definitions be such memory Hogs?  I know there are large numbers of variables, involved, but perhaps there is something we can do to reduce the memory load.

it seems to take a long time to open the LokSound 4 definitions too, even on my primary development machine.  Making that more efficient would improve the user experience.